src/main/java/Serf.java

```diff
@@ -0,0 +1,100 @@
+import java.util.Scanner;
+public class Serf {
+    public static void main(String[] args) {
+        String VERTICALLINES = "    ____________________________________________________________";
+        String FIVEWHITESPACES = "     ";
+        Task[] taskList = new Task[100];
+        int taskCounter = 0; // tracks number of tasks in taskList
```

Comment:
Generally code quality is ok. The area of improvement is mainly to create individual functions instead of adding everything in the while loop.

Reply:
Thank you for your comment.